WPTRT/theme-sniffer
# Feature/show codes

fixes #156

This PR adds the PHPCS codes and a copy to clipboard functionality for annotations to the report (not raw).

![image](https://user-images.githubusercontent.com/11907254/55277448-c306ea00-52d6-11e9-9794-9f49324be1e2.png)

To test:
1. Run report on twentyninteen.
2. Ensure error/warning sources are displayed.
3. Tool tip should be displayed on hover "Copy Annotation"
4. Click on item.

The result is that the tooltip should change to say "Copied!" and the annotation is copied to your clipboard, so you can press ctrl+v to easily paste it in code.

Notes:  
- The copy to clipboard functionality is the same lib used in Gutenberg.
- Since there was more discussion in #156 about the manually added items by Theme Sniffer, there's some refactoring to be done to generate error/warning source codes for those items.  I think this would be better suited for a new ticket and leave the scope down to just this feature.  For the time being, items without error/warning source codes do not display this feature/row.
